ui/src/app/applications/components/application-details/application-details.scss

```diff
@@ -258,19 +260,21 @@ $header: 120px;
             padding-bottom: 6px;
         }
 
-        .zoom-value {
+        .panel-options-text {
```

Comment:
I think this change affects the zoom field of the floating tool bar.  When the zoom is changed from 100% to 90%, and therefore, there is less space of 'text' to fill in the text field, the text field should not shrink in width. And conversely, if you increase the zoom from 90% to 100%, the text field field should remain constant.  Compare the behaviour of this before and after